src/azure-cli-core/HISTORY.rst

```diff
@@ -5,7 +5,7 @@ Release History
 
 2.58.0
 ++++++
-* [BREAKING CHANGE] `az login`: Use WAM as the default authentication method on Windows. If you encounter any issue and want to opt out, run `az config set core.enable_broker_on_windows=false`, `az account clear` and `az login` (#28085)
+* [BREAKING CHANGE] `az login`: Replace `allow_broker` with `enable_broker_on_windows` (#27726)
```

Comment:
As WAM is not enabled by default now, we should announce `allow_broker`'s renaming.

It turns out splitting #27726 and #28085 is a correct decision, as it makes reverting #28085 easier.